ui-v2/app/templates/dc/services/show.hbs

```diff
@@ -31,7 +30,7 @@
                 (hash label="Upstreams" href=(href-to "dc.services.show.upstreams") selected=(is-href "dc.services.show.upstreams"))
 '')
                 (hash label="Instances" href=(href-to "dc.services.show.instances") selected=(is-href "dc.services.show.instances"))
-(if (not item.Service.Kind)
+(if (not-eq item.Service.Kind 'terminating-gateway')
```

Comment:
This means that we can see the intentions for `ingress-gateways` which have the same relationship as normal service to service intentions. Support for the slightly different `terminating-gateways` are to be added soon. `mesh-gateways` never have intentions or tabs on this page.